biologic_mpr.yml

```diff
@@ -0,0 +1,17 @@
+id: >-
+  biologic-mpr
+name: >-
+  Biologic MPR file
+description: >-
+  A binary data file created by Biologic cyclers, containing time- and frequency-domain data pertaining to many electrochemical techniques, such as voltammetry, potentiometry, coulometry and impedance spectroscopy.
+associated_vendors: >-
+  - Biologic
+subject: ->
```

Comment:
I'm not clear on the syntax here, but what's the difference between `>-` and `->`?

Reply:
One is a typo, the other isn't :)